### ISSUE TITLE
Krack attacks

### ISSUE THREAD

**Issue by pparent76** [Nossiac/mtk-openwrt-feeds#34]
Am I right to think that this driver is vulnerable to Krack attacks and will always be?

**Comment by maurerr** [comment]
only in repeater/client mode
AP should be safe

**Comment by pparent76** [comment]
Thank's a lot!

Can you explain how can it be safe in AP mode, when it was written before krack attacks were revealed?

**Comment by Nossiac** [comment]
Mediatek has released patches to its vendors to protect against krack attacks.
Not merged here yet... busy days...

**Comment by maurerr** [comment]
read the KRACK documentation - it's not AP related but client.
It's useless to patch your AP if you still use client OS (Android/IOS) unpatched

**Comment by Nossiac** [comment]
Hmm, as I learned, AP still got the chance to detect suspicious attacks and drop the athentication session immediately.

**Comment by pparent76** [comment]
Yes as far I as understand AP and client can and shoud be patched.
for exemple hostapd has been patched.
